### PR TITLE
Expanded web3config Guide

### PR DESCRIPTION
**Pull request by MeenuyD**
Expanded web3config Guide in Documentation: Added explanations and code samples for handleRevert and defaultAccount parameters.
#6805

### CODE REVIEW

File: docs/docs/guides/web3_config/index.md [hunk 1]
```diff
@@ -30,6 +30,55 @@ There is list of configuration params that can be set for modifying behavior of
 
 ## Global level Config
 
+`handleRevert` is a configuration option that specifies whether the web3 provider should handle revert errors automatically. When set to true, the web3 provider will catch revert errors and return a more informative error message instead of throwing an exception.
```
Comment (jdevcs) [inline docs/docs/guides/web3_config/index.md:33]: @MeenuyD Thanks for working on this, I'll suggest to create a new section in this guide, for explaning config options, and we need desc of all of following config params:
image

Comment (jdevcs) [reply docs/docs/guides/web3_config/index.md:33]: & if you open docs for these params in https://docs.web3js.org/guides/web3_config/  there is already desc that can be reused in this new section, 

as second step I'll suggest to add code snippets as well in doc as examples.

Comment (MeenuyD) [reply docs/docs/guides/web3_config/index.md:33]: Ok

Comment (jdevcs) [reply docs/docs/guides/web3_config/index.md:33]: @MeenuyD let us know when updates are complete, thx

**Review by avkos — CHANGES_REQUESTED** [review]
wait for changes

**Comment by jdevcs** [comment]
@MeenuyD

**Pull request closed without merging**